Assets.md

```diff
@@ -0,0 +1,59 @@
+## SVGs
+
+### 1. Export
+
+When we export an icon from Figma or Zeplin, we should export the icon container instead of the actual icon. _Check the image_
+![image](https://user-images.githubusercontent.com/55184443/115124509-edfa2e80-9fc2-11eb-8386-fa29ad4c0548.png)
+In this case, we will export `Menu` as SVG instead of the `Icon` element.
+
+> With this, every icon we export will have the exact box dimensions, which is easier to maintain in the long run.
+
+### 2. Optimization
+
+Before adding a new icon to our codebase, we should optimize it. [`svgo`](https://github.com/svg/svgo) is a go-to tool for this purpose, with [SVGOMG](https://jakearchibald.github.io/svgomg/) as its hosted UI. [Here are the preferred settings](https://gist.github.com/kristian240/bf7be2570e7cc8074718484130d5ae2e) that should give the optimal output.
+
+### 3. Add to codebase
+
+After we exported and optimized the icon, we can add it to our codebase. We will add this new icon in one of two directories:
+
+#### a. `src/assets/icons`
+
+Before using this a SVG icon this way, few things are mandatory:
+
+- Make sure that the SVG element has only the `viewBox` property (without `width` and `height`). The context surrounding the icon should define its dimensions.
+- Sometimes, designers use `<mask>` element in SVGs. Those elemens are referenced by ID and if you use multiple SVGs per HTML document, mulitple `mask` elements will have same ID and as a result, SVGs won't work as expected. Often these `mask` elements are simple to resolve by hand so make sure to remove them.
```

Comment:
```suggestion
- Sometimes, designers use the `<mask>` element in SVGs. Those elements are referenced by an ID, and if you use multiple SVGs per an HTML document, multiple `mask` elements will have the same ID. As a result, SVGs won't work as expected. Often these `mask` elements are simple to resolve by hand so make sure to remove them.
```
-----
>Often these `mask` elements are simple to resolve by hand so make sure to remove them.

This is a bit misleading. Should they be removed or resolved in some other way?